Resources/Prototypes/Catalog/Jukebox/Standard.yml

```diff
@@ -228,7 +228,7 @@
 
 - type: jukebox
   id: Impossible
-  name: Darren Knob - Impossible
+  name: Darren Korb - Impossible
```

Comment:
ive been thinking of making a pr to fix this for months. thank you.

Reply:
its honestly insanely embarrassing that i did this twice lmao